### DISCUSSION TITLE
How to reduce the FP of background prediction in YOLOv5s

### DISCUSSION THREAD

**Discussion by Ahmed-Sharba** [community/community#80910] [Programming Help]
### Body

Hi everyone I trained a dataset of two classes (vehicles and planes) with 2776 images including 249 background images in training, 781 images including 69 background images in validation, and 391 images including 32 background images in the test. The result is very good but the background in the confusion matrix is divided between two classes such as the picture uploaded. How can I prevent the FP of the background in the confusion matrix?
![confusion_matrix](https://github.com/community/community/assets/154312990/0b12c680-eede-4d57-b397-42ec43d00810)

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by BhathikaGimhan** [comment]


**Reply by Akash1134** [reply]
Hi @BhathikaGimhan  , 

We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.
